setup.cfg

```diff
@@ -1,3 +1,16 @@
 [bdist_wheel]
 # code is written to work on both Python 2 and Python 3
-universal=1
+universal = 1
+
+[pep8]
+max-line-length = 119
+# Ignore E127: checked by pylint
+# E127 continuation line over-indented for visual indent
+ignore = E127
+exclude = migrations,setup.py
+
+[pydocstyle]
+match-dir = (?!tests|migrations|\.).*
+match = (?!setup).*.py
+add-select = D203,D404
```

Comment:
We had more docstrings in D203. Personally I like the symmetry of one extra line before and one after the class doctoring (the one after the docstring is required). Whereas in functions there are no lines before and after the docstring, so there is an inherent symmetry there. I do not mind switching if you would prefer that.